css/amp.css

```diff
@@ -146,7 +146,12 @@ i-amp-sizer {
   /* Just state. */
 }
 
+
+/** Only show the last placeholder element */
 .-amp-element > [placeholder] {
+  display: none;
+}
+.-amp-element > [placeholder]:last-of-type {
```

Comment:
Ok after tons of experiments it seems it is impossible to do last-match in CSS as we want here. I came up with a different idea of how to approach this. Still in CSS.

Instead of hiding placeholders we just show all placeholders with trbl=0 as it's currently is. The only difference is that we set a background for the placeholders (e.g. `#efefef`), this will force the last placeholder to show at the top and virtually hide all the other placeholders.

The other change that we'd need to do is with `togglePlaceholder` which now needs to toggle ALL placeholders instead of just the one. I don't see any problem with this.

I'll update the code to reflect this but if there are any concerns with this please let me know.